nicenumber/nicenumber.py

```diff
@@ -46,3 +46,30 @@ def to_numeric(string:str, family:str = 'number'):
     """
 
     return
+
+def to_pandas(df, col=None, type='human', family='number'):
```

Comment:
@camharris22 I'd change "type" to "type_" or just a different word all together as "type" is a reserved keyword in python!

Reply:
Also I'd add type hints eg df : pd.DataFrame, col : Union[str, list], family: str etc